plugin/src/openshift/components/KialiController.tsx

```diff
@@ -79,11 +79,7 @@ type KialiControllerProps = KialiControllerReduxProps & {
   children: React.ReactNode;
 };
 
-const defaultPluginConfig: PluginConfig = {
-  graph: {
-    impl: 'pf'
-  }
-};
+const defaultPluginConfig: PluginConfig = {};
```

Comment:
@josunect should we add the observability default config here as well?
```suggestion
const defaultPluginConfig: PluginConfig = {
  "observability": {
    "instance": "sample",
    "namespace": "tempo",
    "tenant": "default"
  }
}
```